tests/test_freetextsearch.py

```diff
@@ -153,54 +1466,50 @@ def testBestHitSearchFieldsTarget(self):
         
         self.assertEqual(len(json_response['data']),6)
         
-        braf_data = json_response['data'][0];
-        self.assertEqual( braf_data['highlight'], '')
+        braf_data = json_response['data'][0]
```

Comment:
in the besthitsearch the highlights are being parsed in the call to assess wether it is an exact match. The highligts here are a subset of what is in the search. Happy to disable them completely in this search